docs/messaging-service/outgoing-api-spec.md

```diff
@@ -0,0 +1,53 @@
+# Outgoing API specification
+
+Here you can see the example request bodies that messaging service can execute when you configure it to use Custom API integrations.
+
+## SMS REST integration
+
+If you choose the Custom API integration option for your SMS gateway, the Messaging service will forward the actual message to the specified endpoint by doing
+an HTTP POST request. An example request body that will be sent, will look like:
+
+```json
+{
+  "from": "+987654321",
+  "to": "+123456789",
+  "renderedTextBody": "Hello from service John!",
+  "contentParameters": {
+    "name": "John",
+    "lastName": "Doe"
+  },
+  "messageId": "5f02a5c0-ed54-4e09-a41e-620e1296d3ad",
+  "userId": "dc36b80e-08d3-40c6-8b1d-db5b2ae4aa4e"
+}
+
+```
+
+## E-mail REST integration
+
+If you choose the Custom API integration option for your email gateway, the Messaging service will forward the actual message to the specified endpoint by doing
+an HTTP POST request. An example request body that will be sent, will look like:
+
+```json
+{
+  "fromAddress": "service@onewelcome.com",
+  "toAddres": "john.doe@example.org",
```

Comment:
Is this a typo in the documentation, or also in the implementation? It should be `toAddress`.

Reply:
only in docs, bad copy-paste of field name.

Reply:
fixed